src/Caching/StackExchangeRedis/src/RedisCache.cs

```diff
@@ -284,7 +299,7 @@ private byte[] GetAndRefresh(string key, bool getData)
                 return results[2];
             }
 
-            return null;
+            return EmptyByteArray;
```

Comment:
@Tratcher this seems like the only sensible option without external changes but it's a breaking change?

Or should we ask to change dotnet/runtime's definition to byte[]? https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Caching.Abstractions/src/IDistributedCache.cs#L14-L19

Reply:
The API docs are clear that it returns null if the entry is not found. That's an important distinction because an empty entry and a missing entry can be semantically different.

Let's get the nullability fixed on IDistributedCache.

Reply:
If there is anything I can do to assist here I am game/able.

Reply:
I filed: https://github.com/dotnet/runtime/issues/52148, we'll need that before we can enable nullability properly on this PR.